code/server/views/base.jade

```diff
@@ -32,7 +32,7 @@ html(lang='en')
                 span.icon-bar
                 span.icon-bar
                 span.icon-bar
-            a.navbar-brand#home(href='/') Gradr
+            a.navbar-brand#home(href='/main') Gradr
```

Comment:
I kept forgetting to make this change, it makes me really happy that clicking Gradr won't tell me to log in anymore.